mbus/mbus-protocol.h

```diff
@@ -554,6 +554,8 @@ const char *mbus_data_fixed_function(int status);
 long        mbus_data_record_storage_number(mbus_data_record *record);
 long        mbus_data_record_tariff(mbus_data_record *record);
 int         mbus_data_record_device(mbus_data_record *record);
+const char *mbus_data_record_unit(mbus_data_record *record);
+const char *mbus_data_record_value(mbus_data_record *record);
```

Comment:
Is this related to cmake support?

Reply:
No, its something that we needed, can be done a future commit to have a clean commit